webui/.prettierrc.json

```
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://json.schemastore.org/prettierrc",
+  "semi": false,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "printWidth": 100,
+  "trailingComma": "none"
```

```suggestion
  "semi": true,
  "tabWidth": 4,
  "singleQuote": false,
  "printWidth": 120,
  "trailingComma": "none"
```

for consistency with our Python code